tests/custom/converter/test_export_yolo.py

```diff
@@ -66,7 +67,23 @@ def create_temp_folder():
     shutil.rmtree(temp_dir)
 
 
-def test_convert_to_yolo(create_temp_folder):
+def test_convert_to_yolo_with_images(create_temp_folder):
```

Comment:
I'd like to see a testcase for without images as well - could we maybe use pytest parametrize to cover both?

Reply:
Fixed